west.yml

```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: ad142f5612d927e29b1f9606e8edade871b8a526
+      revision: pull/290/head
```

Comment:
Is here should be a commit num?

Reply:
Yes, it should be, but for testing we can add a pull request until that PR is merged.

This PR has a DNM (do not merge) label and it cannot be merged until this label is removed (this is don't automatically). This label is added, also automatically, because we have a PR in west.yml, instead of a commit id.

![image](https://github.com/zephyrproject-rtos/zephyr/assets/74421316/4eae7390-91c7-4f54-a5d6-aa78d6bb825f)